components/Card.js

```diff
@@ -3,11 +3,13 @@ import styled from "styled-components";
 import { EntertainmentIcon } from "@/components/Icons";
 import { AppliancesIcon } from "@/components/Icons";
 import { WorkIcon } from "@/components/Icons";
+import { LightingIcon } from "@/components/Icons";
 
 const CATEGORY_MAP = {
   Entertainment: <EntertainmentIcon />,
   Appliances: <AppliancesIcon />,
   Work: <WorkIcon />,
+  Lighting: <LightingIcon />,
 };
 
 const Wrapper = styled.li`
```

Comment:
i think to use an li element is semanticly not correct and the Wrapper should be named StyledWrapper maybe it is better to use the Fragment tag

Reply:
Thank you for your comment, the wrapper is a list item because it is an item in an unordered list.

Reply:
One big disadvantage of having your component wrapper be a list item is that your component can only be used correctly inside a list. That works strongly against the reusability of your component. You could instead add the list items in the component / page where you use this Card component. Like this:
```
<ul>
   {cards.map(card => (<li key={card.id})><Card ... /></li>))}
</ul>
```

Reply:
Thank you for your comment, implemented.